docs/developers/objects.md

```diff
@@ -55,7 +55,19 @@ These are used when deciding if a run has been uploaded, partially uploaded, is
 
 The `directory` field holds the path to the directory of some potential sequencing run.
 
-`"new"`, `"partial"`, `"complete"`, and `"invalid"` are valid in the `status` field.
+The `status` field can hold the following options, defined in the file. (e.g. DirectoryStatus.NEW)
+
+* NEW
+    * New runs, ready to upload
+* INVALID
+    * Used when a run directory does not have the base requirements to act as a run
+    * Never written to a status file
```

Comment:
Can you expand on what you mean about `Never written to a status file`?  I get what you mean but only because you explained it to me.

Reply:
Added